RequestManagementSystem/Service/ReqProxyHandler.py

```diff
@@ -109,7 +109,7 @@ def sweeper( cls ):
           requestString = "".join( open( cachedFile, "r" ).readlines() )
           cachedRequest = eval( requestString )
```

Comment:
Hum, I just realized now... I think that I did not adapt the ReqProxyHandler to the new json system. So not sure that this fix will make it work